### PR TITLE
module: add images to system.build

### PR DESCRIPTION
**Pull request by Lassulus**

### CODE REVIEW

File: module.nix [hunk 1]
```diff
@@ -71,6 +71,13 @@ in
       disko = builtins.trace "the .disko output is deprecated, please use .diskoScript instead" (cfg.devices._scripts pkgs).diskoScript;
       diskoNoDeps = builtins.trace "the .diskoNoDeps output is deprecated, please use .diskoScriptNoDeps instead" (cfg.devices._scripts pkgs).diskoScriptNoDeps;
 
+      diskoImages = diskoLib.makeDiskImage {
```
Comment (phaer) [inline module.nix:74]: Is it intentional that the binding uses plural form (image**s**) while the function uses singular form?

Comment (Lassulus) [reply module.nix:74]: no :D but which one is better?

Comment (Lassulus) [reply module.nix:74]: it returns a path to a directory with possibly multiple imnages

Comment (phaer) [reply module.nix:74]: Is it too late yet to rename the function to plural form as well then? I don't image their being too many users yet, but what do i know 😅 an alias could be a good compromise if so?

Comment (Mic92) [reply module.nix]: It doesn't had documentation so far. So I think it is rather safe.

**Review by phaer — APPROVED** [review]
I think this is fine? Just letting you decide when to press that button :)

**Comment by Lassulus** [comment]
@mergify queue

**Pull request merged**